source/CommonJobs/CommonJobs.Application.EvalForm/Commands/DeleteEvaluationCommand.cs

```diff
@@ -46,5 +50,11 @@ public static string Escape(string value)
         {
             return "\"" + value.Replace("\\", "\\\\").Replace("\"", "\\\"") + "\"";
         }
+
+        private bool IsClosedPeriod(string period)
+        {
+            // TODO: Remove hardcoding and find a way to determinate if a period is closed/finalized
```

Comment:
I am not sure if it should be the criteria to allow to delete periods, but it does not care.